tests/python_tests/test_sampling.py

```diff
@@ -317,6 +317,64 @@ def test_individual_generation_configs_random(tmp_path, test_struct: RandomSampl
     # Discrete_distribution impl depends on platform, model inference results may depend on CPU.
 
 
+@pytest.mark.precommit
+@pytest.mark.parametrize("sampling_config", [get_greedy(), get_beam_search(), get_multinomial_all_parameters()])
+@pytest.mark.parametrize("max_num_batched_tokens", [2, 4, 256])
+def test_echo_without_completion(tmp_path, sampling_config, max_num_batched_tokens):
+    generation_config = sampling_config
+    generation_config.max_new_tokens = 0
+    generation_config.echo = True
+
+    scheduler_config = get_scheduler_config()
+    scheduler_config.max_num_batched_tokens = max_num_batched_tokens
+    generation_configs = [generation_config]
+    model_id : str = "facebook/opt-125m"
+    model, hf_tokenizer = get_model_and_tokenizer(model_id, use_optimum=True)
+
+    model_path : Path = tmp_path / model_id
+    save_ov_model_from_optimum(model, hf_tokenizer, model_path)
+
+    pipe = ContinuousBatchingPipeline(model_path.absolute().as_posix(), Tokenizer(model_path.absolute().as_posix(), {}), scheduler_config, "CPU", {})
+
+    outputs = pipe.generate(["What is OpenVINO?"], generation_configs)
+    assert(len(outputs))
+    for output in outputs:
+        assert(len(output.m_generation_ids))
+        for sequence in output.m_generation_ids:
+            assert(sequence == "What is OpenVINO?")
+
+    del pipe
+    shutil.rmtree(model_path)
```

Comment:
You've already requested `tmp_path` from pytest. Why clean in manually?

Reply:
I was inspired by: https://github.com/openvinotoolkit/openvino.genai/blob/master/tests/python_tests/common.py#L339
I thought that even if I don't use high level methods from `common` I should follow their flow.